.github/workflows/build-and-test.yml

```diff
@@ -12,5 +12,5 @@ jobs:
     - name: Build and run fingerprint tests
       run: |
         docker run --rm -v ${{ github.workspace }}:/root/models -u "$(id -u):$(id -g)" \
-        omnetpp/inet:o5.5.1-3.6.6 \
+        omnetpp/inet:o5.5.1-4.1.1 \
```

Comment:
We would need a Docker container with Omnet 5.5.1 and INET 4.1.1.  Seems that guessing the name does not work - what would be the correct name?

Reply:
The omnetpp/inet:o5.5.1-4.1.1 image has now been built.
Any references to the /root/inet/ directory in it should be updated to /root/inet4/ for the new image.

Reply:
Or, if that would be too much trouble (an inconvenient difference from the development setups); perhaps a soft or hard link named `inet` could be created to `inet4` in the container as an early step of the test.

Reply:
Inside the top level makefile, you should change `-KINET_PROJ=../../inet` to `-KINET_PROJ=../../inet4`

Reply:
Thanks for setting up the container - changed the path to the INET project to ../inet4. Now the project is found.

Reply:
Fingerprint tests were failing since we used a more current version of INET4 in our CI. However, we have recalculated the fingerprints based on INET4 Tag v4.1.1 now and tests pass.
@rhornig Regarding the mistakenly committed files you were right, of course these should never have been committed. Have been removed, thanks for noticing.